src/main/kotlin/com/example/minibank/controllers/AuthController.kt

```diff
@@ -56,17 +79,32 @@
     @DeleteMapping("/user/{id}")
     @Operation(summary = "Удаление пользователя")
     fun delete(@CookieValue("jwt") jwt: String?, @PathVariable id: Int): ResponseEntity<Any> {
+
```

Comment:
🚫 **[ktlint]** <standard:no-empty-first-line-in-method-block> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>First line in a method block should not be empty